simpleflow/swf/process/worker/base.py

```diff
@@ -47,13 +53,17 @@ def __init__(self, domain, task_list, heartbeat=60):
         :type task_list:
         :param heartbeat:
         :type heartbeat:
+        :param process_mode: Whether to process locally (default) or spawn a Kubernetes job.
+        :type process_mode: str
         """
         self.nb_retries = 3
         # heartbeat=0 is a special value to disable heartbeating. We want to
         # replace it by None because multiprocessing.Process.join() treats
         # this as "no timeout"
         self._heartbeat = heartbeat or None
 
+        self.process_mode = 'kubernetes' if process_mode == 'kubernetes' else 'local'
```

Comment:
`self.process_mode = process_mode or 'local'`?

Reply:
I wanted to specifically check that we get either "kubernetes" or "local" for now.

Maybe I could be more explicit?

```python
# somewhere.py
VALID_PROCESS_MODES = [
    "local",
    "kubernetes",
]

# worker/base.py
self.process_mode = process_mode or "local"
assert self.process_mode in VALID_PROCESS_MODES
```
?

Reply:
Yay for useful error messages, I agree!

Reply:
Oh, and a set? I get the "pedantic" wrapping tags now!